src/LiveComponent/src/LiveComponentHydrator.php

```diff
@@ -410,7 +410,7 @@ private function dehydrateObjectValue(object $value, string $classType, ?string
         return $dehydratedObjectValues;
     }
 
-    private function hydrateValue(mixed $value, LivePropMetadata $propMetadata, object $parentObject): mixed
+    public function hydrateValue(mixed $value, LivePropMetadata $propMetadata, object $parentObject): mixed
```

Comment:
We should add a phpdoc method description to this and it should be moved up above the private methods